src/adapter.ts

```diff
@@ -7,7 +7,13 @@ import {
 } from "@algolia/client-search";
 import { ItemsJsOptions, SearchClient } from "./itemsjsInterface";
 
-export function getSearchClient(index: any): SearchClient {
+let index;
+
+export function getSearchClient(newIndex?: any): SearchClient {
+  // This newIndex provides the possitblity to not use the latest created index.
+  // Added to make it backwords compatiable with v1.0.4
+  if (newIndex) index = newIndex;
```

Comment:
Ik vind deze wat moeilijk te lezen, kun je niet performSearch(newIndex || index) gebruiken?

Reply:
Didn't know this was possible, indeed an easier fix!